src/Form.jsx

```diff
@@ -145,7 +145,7 @@ function Form({
                     setOptions: updateFieldOptions(stepsArg)
                 });
             }
-            clientArg.registerEvent(newKey, 'load');
+            clientArg.registerEvent(newKey, '', 'load');
```

Comment:
typo

Reply:
No peter, it is an empty string.

Reply:
in that case should we make load the second argument and have empty string be the default arg for next_step_key

Reply:
Updated